src/osd/PrimaryLogPG.cc

```diff
@@ -5494,9 +5494,9 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
   object_info_t& oi = obs.oi;
   const hobject_t& soid = oi.soid;
   bool skip_data_digest = osd->store->has_builtin_csum() &&
-    g_conf->get_val<bool>("osd_skip_data_digest");
-  auto osd_max_object_size = cct->_conf->get_val<uint64_t>(
-    "osd_max_object_size");
+    cct->_conf->osd_skip_data_digest;
+  const uint64_t osd_max_object_size = \
```

Comment:
why newline?

Reply:
I've though we would cross the 80th column boundary otherwise. Fortunately, just mislooked. Thanks!